apps/src/bounce/skins.js

```diff
@@ -54,6 +72,31 @@ var CONFIGS = {
       'Utah Jazz',
       'Washington Wizards',
     ],
+  },
+
+  sports: {
+    drawTiles: false,
+    backgrounds: [
+      'basketball',
+      'football',
+      'hockey',
+      'soccer'
+    ],
+    balls: [
+      'basketball',
+      'football',
+      'hockey',
+      'soccer'
+    ],
+    paddles: [
+      'hand_1',
+      'hand_2',
+      'hockey',
+      'soccer'
```

Comment:
how difficult would it be to name these something like "foot" and "hockey_stick", for consistency?